### ISSUE TITLE
Generated image in Stable Diffusion doesn't look like sample generated by kohya_ss

### ISSUE THREAD

**Issue by NeyaraIA** [bmaltais/kohya_ss#1627]
When I start training my LORA, I generate samples. These are pretty good, but when I test in stable diffusion, I don't get the same result, the Lora is almost not present. I use the same model as during training, with the same prompt, cfg, etc... Could anyone give me some advice?

**Comment by v0xie** [comment]
If you're using Automatic, try enabling the setting in Compatibility section: `Lora/Networks: use old method that takes longer when you have multiple Loras active and produces same results as kohya-ss/sd-webui-additional-networks extension`

**Comment by dgr9527** [comment]
I had the same problem @kohya-ss

**Comment by dgr9527** [comment]
> If you're using Automatic, try enabling the setting in Compatibility section: `Lora/Networks: use old method that takes longer when you have multiple Loras active and produces same results as kohya-ss/sd-webui-additional-networks extension`

I tried. It didn't work

**Comment by v0xie** [comment]
> > If you're using Automatic, try enabling the setting in Compatibility section: `Lora/Networks: use old method that takes longer when you have multiple Loras active and produces same results as kohya-ss/sd-webui-additional-networks extension`
> 
> I tried. It didn't work

Can you post your training configuration file here so we can see what settings you're using to train?

**Comment by dgr9527** [comment]
> > > If you're using Automatic, try enabling the setting in Compatibility section: `Lora/Networks: use old method that takes longer when you have multiple Loras active and produces same results as kohya-ss/sd-webui-additional-networks extension`
> > 
> > 
> > I tried. It didn't work
> 
> Can you post your training configuration file here so we can see what settings you're using to train?

 v2=False,
 v_parameterization=False,
 pretrained_model_name_or_path='sd1.5.ckpt',
 tokenizer_cache_dir=None,
 train_data_dir=None,
 shuffle_caption=False,
 caption_extension='.caption',
 caption_extention=None,
 keep_tokens=0,
 caption_prefix=None,
 caption_suffix=None,
 color_aug=False,
 flip_aug=False,
 face_crop_aug_range=None,
 random_crop=False,
 debug_dataset=False,
 resolution='512,512',
 cache_latents=True,
 vae_batch_size=1,
 cache_latents_to_disk=False,
 enable_bucket=True,
 min_bucket_reso=256,
 max_bucket_reso=1024,
 bucket_reso_steps=64,
 bucket_no_upscale=True,
 token_warmup_min=1,
 token_warmup_step=0,
 dataset_class=None,
 caption_dropout_rate=0.0,
 caption_dropout_every_n_epochs=0,
 caption_tag_dropout_rate=0.0,
 reg_data_dir=None,
 in_json=None,
 dataset_repeats=1,
 output_dir='./lora_model',
 output_name='101225',
 huggingface_repo_id=None,
 huggingface_repo_type=None,
 huggingface_path_in_repo=None,
 huggingface_token=None,
 huggingface_repo_visibility=None,
 save_state_to_huggingface=False,
 resume_from_huggingface=False,
 async_upload=False,
 save_precision='fp16',
 save_every_n_epochs=5,
 save_every_n_steps=None,
 save_n_epoch_ratio=None,
 save_last_n_epochs=None,
 save_last_n_epochs_state=None,
 save_last_n_steps=None,
 save_last_n_steps_state=None,
 save_state=False,
 resume=None,
 train_batch_size=1,
 max_token_length=None,
 mem_eff_attn=False,
 xformers=False,
 sdpa=False,
 vae=None,
 max_train_steps=16000,
 max_train_epochs=None,
 max_data_loader_n_workers=8,
 persistent_data_loader_workers=False,
 seed=None,
 gradient_checkpointing=False,
 gradient_accumulation_steps=1,
 mixed_precision='fp16',
 full_fp16=False,
 full_bf16=False,
 clip_skip=None,
 logging_dir='./lora_model/log',
 log_with=None,
 log_prefix=None,
 log_tracker_name=None,
 log_tracker_config=None,
 wandb_api_key=None,
 noise_offset=None,
 multires_noise_iterations=None,
 ip_noise_gamma=None,
 multires_noise_discount=0.3,
 adaptive_noise_scale=None,
 zero_terminal_snr=False,
 min_timestep=None,
 max_timestep=None,
 lowram=False,
 sample_every_n_steps=None,
 sample_every_n_epochs=None,
 sample_prompts=None,
 sample_sampler='ddim',
 config_file=None,
 output_config=False,
 metadata_title=None,
 metadata_author=None,
 metadata_description=None,
 metadata_license=None,
 metadata_tags=None,
 prior_loss_weight=1.0,
 optimizer_type='DAdaptAdam',
 use_8bit_adam=False,
 use_lion_optimizer=False,
 learning_rate=0.0005,
 max_grad_norm=1.0,
 optimizer_args=None,
 lr_scheduler_type='',
 lr_scheduler_args=None,
 lr_scheduler='cosine',
 lr_warmup_steps=0,
 lr_scheduler_num_cycles=1,
 lr_scheduler_power=1,
 dataset_config='./dataset_1012666.toml',
 min_snr_gamma=None,
 scale_v_pred_loss_like_noise_pred=False,
 v_pred_like_loss=None,
 weighted_captions=False,
 no_metadata=False,
 save_model_as='safetensors',
 unet_lr=0.0005,
 text_encoder_lr=0.0005,
 network_weights=None,
 network_module='networks.lora',
 network_dim=8,
 network_alpha=1,
 network_dropout=None,
 network_args=None,
 network_train_unet_only=False,
 network_train_text_encoder_only=False,
 training_comment=None,
 dim_from_weights=False,
 scale_weight_norms=None,
 base_weights=None,
 base_weights_multiplier=None,
 no_half_vae=False,
 num_cpu_threads_per_process=8
